runner_option.go

```diff
@@ -20,11 +20,16 @@ type httpRunnerConfig struct {
 	SkipVerify           bool   `yaml:"skipVerify,omitempty"`
 	Timeout              string `yaml:"timeout,omitempty"`
 	UseCookie            *bool  `yaml:"useCookie,omitempty"`
-	Trace                *bool  `yaml:"trace,omitempty"`
+	Trace                traceConfig
```

Comment:
How can I make the nested traceConfig type and the original non-nested bool type compatible with each other?

Reply:
Looks like we need to implement MarshalYAML as well as MarshalJSON of JSON

Reply:
Thanks for the advice!
It took a lot of work, but I did it.
https://github.com/k1LoW/runn/pull/742/commits/597dc8c8218ee226f398133e8c29a9cbbcd84a9f